app/app.css

```diff
@@ -23,3 +23,23 @@
   content: "|";
   padding-right: 0.3em;
 }
+
+
+.totalmembers{ text-align:center;}
+.table > thead > tr> th{text-align: center;}
+.table > tbody > tr> td{text-align: center;}
+.btn.btn-default.btn-block{text-align: center;}
+
+.membersbound {background: #fff; color: #373737;}
+.membersbound > .row > .col-sm-5 > div {display: none; padding: 20px 25px 5px;}
```

Comment:
Layered CSS is generally bad practice, create a single class and apply accordingly.